source/style-guide/style/punctuation.txt

```diff
@@ -138,7 +138,7 @@ in the *Chicago Manual of Style*.
        Unlike the other alarms in this list you set the network check
        alarm variable upon network check creation.
 
-       To learn more, see Upgrading Ops Manager.
+       To learn more see Upgrading Ops Manager.
```

Comment:
[tiny nit]

Very tiny quibble but I think we should retain the comma here.

Reply:
I did the same thing!